.circleci/signer/sign_image.py

```diff
@@ -0,0 +1,338 @@
+import subprocess
```

Comment:
I would recommend putting this in `ops/signer` - this is where other CI related code lives

Reply:
done